test_matrix.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from matrix import Matrix
```

Comment:
There should be blank line between them.
Notice conventions about imports:
---built in imports---
__blank line__
--third part imports--
__blank line__
--local methods--
__blank line__